### ISSUE TITLE
the schemaless external function will register an error if shema is not configured

### ISSUE THREAD

**Issue by tansuozhe123** [lf-edge/ekuiper#2337]
<img width="542" alt="19982957c687da0e57ffb98216891db" src="https://github.com/lf-edge/ekuiper/assets/25733524/ddd4e80e-b5fd-43a6-a20e-84f038bb6778">
<img width="518" alt="c19e035875e59fcec2610a87c0c06c0" src="https://github.com/lf-edge/ekuiper/assets/25733524/238b09cc-e159-4bfa-b779-af4f57a1df2d">

**Comment by ngjaying** [comment]
Yes, service definition file is still needed. https://ekuiper.org/docs/zh/latest/extension/external/external_func.html#%E9%85%8D%E7%BD%AE

**Comment by tansuozhe123** [comment]
<img width="872" alt="微信图片_20231019091016" src="https://github.com/lf-edge/ekuiper/assets/25733524/e47363c4-d007-4c64-8281-1107110b2f2d">

**Comment by ngjaying** [comment]
Notice that, this feature is in the latest pre release 1.12.0-alpha.2 only.

**Comment by tansuozhe123** [comment]
Ok, I see
